public/Stations.php

```diff
@@ -82,4 +82,16 @@ public function getRelatedOnramps($id) {
   public function getDetectors($id) {
     return Detector::fetchForStation($id);
   }
+
+  /** Will take a coordinate and return the closest stationid
+   *
+   * Will take a coordinate and return the closest stationid
+   *
+   * @param $coord
+   * @return null
+   */
+  public function getStationfromCoord($coord){
+    $s = OrderedStation::fetchAll();
+    return null;
```

Comment:
This is left over from my attempts at getting a station object within PHP instead of the database, I deleted what was there and I guess I forgot the fetching bit by accident. Ill delete it.